nixos/lib/make-disk-image.nix

```diff
@@ -447,8 +447,7 @@ let
     mkdir -p $root
 
     # Copy arbitrary other files into the image
-    # Semi-shamelessly copied from make-etc.sh. I (@copumpkin) shall factor this stuff out as part of
-    # https://github.com/NixOS/nixpkgs/issues/23052.
+    # Semi-shamelessly copied from make-etc.sh.
```

Comment:
Let’s not remove the pointer to the issue here, this stuff is indeed a mess that should be refactored, although #324817 might be more relevant these days.

Reply:
I totally agree that it's a mess (and did some work to clean-up image recently-ish with system.build.image and nixos-rebuild build-image), but I don't see how a reference to a 8 year old stale-issue helps here tbh

Reply:
I just put it back :)

Reply:
You proposed it as a change below at 16:41 and pushed to the branch at a minute or two later... and you did not even bother to remove the "I (@copumkin)" part in it or replace the 8 year old stale issue by the contemporary alternative proposed by @emilazy above.

Behaviour like this actively deters people from contributing.